src/coq_elpi_builtins.ml

```diff
@@ -665,11 +667,12 @@ be distinct).|};
              Evd.check_univ_decl ~poly:false evd UState.default_univ_decl in
           Declare.definition_entry
```

Comment:
Are you saying that this line is supposed to pass false? Or is it inherited from the check_univ_decl above?

Reply:
Yes it is inherited from the check_univ_decl, after the Coq PR you cannot omit `poly` anymore, it is mandatory in all the places it is needed.